doc/classes/Engine.xml

```diff
@@ -118,6 +118,13 @@
 				Returns [code]true[/code] if the game is inside the fixed process and physics phase of the game loop.
 			</description>
 		</method>
+		<method name="get_physics_interpolation_fraction" qualifiers="const">
+			<return type="float">
+			</return>
+			<description>
+				Returns the fraction through the current physics tick we are at the time of rendering the frame. Allows fixed timestep interpolation.
```

Comment:
I suggest a small wording change which sounds more natural to me :slightly_smiling_face:

```suggestion
				Returns the fraction through the current physics tick we are at the time of rendering the frame. This can be used to implement fixed timestep interpolation.
```

Reply:
Yup sure, I'd already noticed the spaces missing on my get_physics_interpolation_fraction, so frustrating lol I need to get the clang checking working on my computer before I upload. Or maybe the clang checks should be done first on the changed files before the rest of the automated builds? :smile:

Will change the wording too.